### ISSUE TITLE
Pressing escape key results in Uncaught TypeError

### ISSUE THREAD

**Issue by tbartel** [angular-ui-tree/angular-ui-tree#695]
Pressing the "Escape" key anywhere on the page results in ui-tree throwing the following error in the console.
```
Uncaught TypeError: Cannot read property 'eventArgs' of undefined
    dragEnd @ angular-ui-tree.js:980
    keydownHandler @ angular-ui-tree.js:1100
    m.event.dispatch @ jquery.js:4670
    r.handle @ jquery.js:4338
```
Recreated here: http://jsfiddle.net/7tom3nbx/1/

**Comment by Till86** [comment]
Got the same issue. Would be nice, if you guys could pull the fix :+1:

**Comment by oliveti** [comment]
I also have the same issue, it is even reproductible in your demo page !

Would be great to merge the fix from @tbartel.

Thank you.

![angularjs ui tree demo 2016-02-25 11-54-02](https://cloud.githubusercontent.com/assets/1235503/13317677/a2822e10-dbb6-11e5-97f2-1183392457e4.png)

**Comment by felixruponen** [comment]
Is this issue still open?

**Comment by SlyNet** [comment]
Still reproduced in 2.15.0 release

**Comment by i-am-dror** [comment]
2.16.0 still has the problem

**Comment by DisasterMan78** [comment]
Also in 2.17.0

**Comment by pmdunggh** [comment]
Line 1065 in v2.17.0 un minified, dragInfo is null when user press Escape key!

**Comment by jacobscarter** [comment]
PR is currently being reviewed for this fix and being tracked in issue #799
